.semgrep/fsm_time.yml

```diff
@@ -30,4 +30,7 @@ rules:
     paths:
       include:
         - "nomad/fsm.*"
```

Comment:
Seems like `fsm.*` has the same problem? 

I don't remember the syntax details, but would something like this work?

```yaml
include:
  - "nomad/fsm*"
  - "nomad/state/state_store*"
exclude:
  - "*_test.go"
```

Reply:
Good shout, modified within: https://github.com/hashicorp/nomad/pull/18315/commits/e49bb26795cf8ac1664580b6edaad570be3f7ac0

Tested locally and it catches the right files.